documentation/asciidoc/topics/proc_exposing_external_service.adoc

```diff
@@ -1,31 +1,83 @@
 [id='exposing_external_services-{context}']
 = Exposing {brandname} to External Clients
-Expose {brandname} clusters to clients running outside {k8s} with external
-services.
+Make {brandname} clusters available to clients running outside {k8s}.
 
 .Procedure
 
-* Specify an external service type with `spec.expose.type` in your
-**Infinispan** CR and then apply the changes.
+. Include `spec.expose` in your **Infinispan** CR to expose {brandname} cluster.
+. Specify one of the following external service types as the value for `spec.expose.type`:
 +
-[source,options="nowrap",subs=attributes+]
-----
-include::yaml_annotated/cr_external_service.yaml[]
-----
+* `LoadBalancer`
+* `NodePort`
+* `Route`
 +
-<1> exposes an external service at port `11222`.
-<2> specifies either a `LoadBalancer` or `NodePort` service resource type.
+. Apply the changes.
 
-//-
+.Load Balancer
 
-`LoadBalancer`:: Use for {k8s} clusters where a load balancer service is
-available to handle external network traffic. You can then use the URL for the
-load balancer service for client connections.
-+
+[NOTE]
+====
 To access {brandname} with unencrypted Hot Rod client connections you must use
 a load balancer service.
+====
+
+[source,options="nowrap",subs=attributes+]
+----
+spec:
+  ...
+  expose:
+    type: LoadBalancer <1>
+----
+
+<1> Exposes {brandname} on the network through a load balancer service.
+
+By default the exposed port is `11222`. You can optionally change the port with
+`spec.expose.nodePort` as in the following example:
+
+[source,options="nowrap",subs=attributes+]
+----
+spec:
+  ...
+  expose:
+    type: LoadBalancer
+    nodePort: 30000
+----
+
+.Node Port
+
+[source,options="nowrap",subs=attributes+]
+----
+spec:
+  ...
+  expose:
+    type: NodePort <1>
+    nodePort: 30000 <2>
+----
+
+<1> Exposes {brandname} on the network through a node port service.
+<2> Defines the port where {brandname} is exposed. If you do not define a port,
+the platform selects one.
+
+.Route
+
+[source,options="nowrap",subs=attributes+]
+----
+spec:
+  ...
+  expose:
+    type: Route <1>
+    host: http://example.org/ <2>
```

Comment:
The same as above about host format

Reply:
This topic should have been deleted. Removed it now.